patients/repo.go

```diff
@@ -1110,7 +1109,7 @@ func (r *repository) TideReport(ctx context.Context, clinicId string, params Tid
 	tags := store.ObjectIDSFromStringArray(*params.Tags)
 
 	if params.LastDataCutoff == nil || params.LastDataCutoff.IsZero() {
-		params.LastDataCutoff = pointer.FromAny(time.Now().UTC().AddDate(0, 0, -5))
+		return nil, errors.New("no lastDataCutoff provided")
```

Comment:
This will result in a 500. Please wrap `errors.BadRequest`

Reply:
wrapped.